notebooks/community/ml_ops/stage6/get_started_with_xgboost_model_online.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<ul><li><strong>Deploying -&gt; Deploy</strong> <code>Model</code> <strong>resources -&gt; resource</strong></li><li>Remove the dot at the end because it's a title</li></ul><blockquote><em>Note:</em> For this example, the deployment container for your <code>AutoML</code> model is predefined. </blockquote><p>Isn't it an XGBoot model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
done

Reply:
<p>resources -&gt; resource :D</p>

Reply:
done